app/js/index.js

```diff
@@ -110,11 +111,11 @@ angular.module('index-app', ['firebase'])
 							team: ['null']
 						});
 					}
+					$scope.username = usersArray.$getRecord(user.uid).name;
```

Comment:
You don't need to submit another pull request. 
Changing code -> commit -> push again
Pull request will update automatically.